app/views/sign_up/emails/show.html.erb

```diff
@@ -38,7 +38,7 @@
   <% if FeatureManagement.enable_load_testing_mode? %>
     <%= link_to(
           'CONFIRM NOW',
-          sign_up_create_email_confirmation_url(confirmation_token: EmailAddress.find_with_email(email).confirmation_token),
+          sign_up_create_email_confirmation_url(confirmation_token: EmailAddress.find_with_email(email)&.confirmation_token),
```

Comment:
I'm trying to wrap my head around what _should_ happen in this case. Your change fixes the immediate bug, and it might be worth getting it out now and then revisiting less immediate stuff.

I don't completely understand why `find_with_email` is returning nil here. Does it have to do with suspension?

I'm almost wondering if we should change line 38 to something like:

```ruby
<% if FeatureManagement.enable_load_testing_mode? && EmailAddress.find_with_email(email) %>
```

and not show the "CONFIRM NOW" link in this case?

Reply:
This option is beneficial as it prevents displaying the "CONFIRM NOW" button when the email is not found.@zachmargolis 

`<% if FeatureManagement.enable_load_testing_mode? && EmailAddress.find_with_email(email) %>`

Reply:
Yeah it makes sense to me to hide the link if we can't actually confirm it